tensorflow_serving/model_servers/main.cc

```diff
@@ -185,11 +185,13 @@ int main(int argc, char** argv) {
   tensorflow::int32 port = 8500;
   bool enable_batching = false;
   tensorflow::string model_name = "default";
+  tensorflow::int32 file_system_poll_wait_seconds = -1;
```

Comment:
I don't think -1 is (or should be) a valid value. Can we make the default 1 instead of -1 please.